### ISSUE TITLE
Blog post: laying out the possibilities

### ISSUE THREAD

**Issue by benmiller314** [benmiller314/cdm2019fall#10]
As you now know, throughout the semester, I will periodically ask you to develop proposals for a collaborative final project that will allow you to integrate – and extend – what you’ve learned about composing digital media. This is the second of those proposals; you can directly reference an idea from [the last one](6), or start fresh. The question is almost the same:

What further possibilities of graphic design might you want to explore in connection with other people, and other media?

As always on the forum, replying directly to each other is optional but encouraged: just use the @ symbol to flag someone by username.

**Comment by emmaknaub** [comment]
I think working with animation for the final project would be cool. We could take the idea of sound layering from "The Magic of Making Sound" that we discussed in the last blog post and layer isolated sounds over animations. Again, I don't know what the specific goal of the project would be, but I think layering sound and animation could leave room for a lot of creative, interesting projects.

**Comment by emaur** [comment]
I completely agree with @emmaknaub. Even before reading her post I instantly thought that creating an animation would combine perfectly with our previous idea of creating sounds ourselves with unique objects. This would allow us to create a story similar to the soundscape project, but now both with sounds and visuals together. By doing this, we are elevating the process we went through with our sound/visual projects to the next level.

**Comment by ajs1102** [comment]
Similar to @emmaknaub  and @emaur  I think animation and sound naturally go together, and would create several possibilities for arguments to be made. Animation could open new ways for sound to embody a physical space, and vice-versa. Another idea I had, which will likely be fleshed out in the next unit, is integrating graphic and audio design into web development. I had this idea since I already have experience working with HTML, CSS, and Javascript, so I already have some ideas in terms of possibilities.

**Comment by fatemaquaid987** [comment]
I agree with the previous posts. Honestly, I was thinking about animation too. However, instead of creating animations on GIMP or photoshop, we could do something more challenging and create simple animations using a programming language like python or javascript.  There is a lot we can do with animations but a fun thing to create would be a simple animated game.  I know, a lot of us are not very familiar with coding and it could be a challenge for many to design and develop a game but since we will be working in groups, we can team up such that every group contains a member who knows how to code.

**Comment by cjc162** [comment]
I also agree animation would be something fun to try. I think it'd be fun to do coding similar to what @fatemaquaid987 mention or making some kind of flip book style animation. We could make many different images with slight variation between them. When "flipped" through, the images could come together to create an animation.

**Comment by wsc9** [comment]
Animation also sounds cool to me. I have never worked with any animating software, so I think it would be cool to try and get into. I also think anything with a video aspect would be interesting. With the rise of social media marketing there are more ways than ever before to get arguments heard and seen.

**Comment by jlatiniii** [comment]
I agree with the previous posts that using animation would be interesting to learn about. We would be combining what we learned with our sound unit and visual unit so it seems like a reasonable next-level project for the end of the semester.

**Comment by brittanybenson8** [comment]
I also think animation sounds cool! I know nothing about animation or how difficult it is (I imagine it is difficult to use), but it might be neat to try to incorporate sound and animate a short little cartoon.

**Comment by ram716** [comment]
I like the flip book idea.. or maybe even an interactive story

**Comment by jadenash21** [comment]
I would like to see a project where we utilize adobe creative suite. I think a lot of us took this class to continue learning about different software that we will actually be using in our post grad careers. I think something with animation or website design would be interesting to put our efforts towards.

**Comment by racheljfu** [comment]
I agree that animation would be interesting, but I also think it would be interesting to do a series of visual rhetoric images that work together as a set (not just like one image). The series could incorporate animation and sound as well.

**Comment by epesetsky** [comment]
I agree with @jadenash21 in that using more Adobe applications could be helpful in future years! Also looking to other comments in this queue, animation could be cool. I've never attempted to do something like it, so learning even just the basics would be new and interesting!

**Comment by kevinikea** [comment]
Make a gif using GIMP?  Might be a bit frustrating but would still be cool to try.
